x-pack/plugins/ml/public/application/explorer/explorer_swimlane.tsx

```diff
@@ -232,13 +295,12 @@ export class ExplorerSwimlane extends React.Component<ExplorerSwimlaneProps> {
     rootParent.selectAll('.lane-label').classed('lane-label-masked', function (this: HTMLElement) {
       return laneLabels.indexOf(d3.select(this).text()) === -1;
     });
-
-    if (swimlaneType === 'viewBy') {
-      // If selecting a cell in the 'view by' swimlane, indicate the corresponding time in the Overall swimlane.
-      this.highlightOverall(times);
-    }
   }
 
+  /**
+   * TODO should happen with props instead of imperative check
```

Comment:
Is this TODO for a follow-up?

Reply:
Yes, some further swim lane component refactoring is required, because at the moment there is some code that affects other instances, which shouldn't be the case.